setup.py

```diff
@@ -123,7 +123,7 @@ def get_data_files(name, version, fullname):  # pylint: disable=R0912
         set_udev_files(data_files)
         set_systemd_files(data_files, dest=systemd_dir_path,
                           src=["init/arch/waagent.service"])
-    elif name == 'coreos':
+    elif name == 'coreos' or name == 'flatcar':  # pylint: disable=R1714
```

Comment:
Can you please resolve this pylint error - 

> # pylint: disable=R1714

Reply:
Done.